docs/staking.md

```diff
@@ -0,0 +1,46 @@
+
+An important characteristic of NPoS is that not all validators participate in the validating process at the same time. Only validators in the *active validator set* perform the above-mentioned operations and earn rewards for doing so. Generally speaking, the set of active validators is limited to a fixed amount of nodes, however in [HydraDX Snakenet](/snakenet) we will pursue a larger set in order to give more people the possibility to participate.
+
+The decision which validators to elect in the set of active validators is made by following the principle of *proportional justified representation*. This principle aims to safeguard decentralization and fair representation by assigning the available slots to validators in proportion to their nominated stake. The higher the amount of staked tokens with a given validator, the higher the chance that the node will be elected in the active set. Validators which are not included in the active set are placed in a waiting list. The set of active validators is updated at the beginning of every era which provides a possible entry window for new validators.
+
+:::note
+
+In a Substrate-based network, time is divided in units called **eras**. In [HydraDX Snakenet](/snakenet), *1 Era = 24 hours*.
+
+:::
+
+Participating as a validator requires a certain level of technical knowledge needed for safely setting up and maintaining a validator node. Misbehavior of a validator node can be punished by slashing, leading to an involuntary loss of funds for you and your nominators. If you think you have the needed experience for running a validator node, you can refer to our [validator guide](/node_setup). Otherwise, we highly recommend that you consider becoming a nominator.
+
+### Nominators
+
+Nominators help secure the network by nominating validators to be elected in the active validator set. They do so by staking their HDX tokens with validators of their choice. The nomination process does not require running and maintaining nodes which makes this form of staking more accessible to people lacking a strong technical background. Tokens which are used to nominate validators are *bonded*, meaning that they are frozen and cannot be used for other purposes. Nominators can always withdraw their nomination which will be reflected at the end of the current era. Nominators can also free up their tokens, however this will become only effective after a waiting period of *28 days* following the unbonding transaction.
```

Comment:
"Nominators can always withdraw their nomination which will be reflected at the end of the current era"

do you mean withdraw their rewards?
or did you mean unbonding
because process is like 
stop nominating
unbond
(wait unbonding period)
withdraw unbonded